.vscode/launch.json

```diff
@@ -9,92 +9,81 @@
       "type": "node",
       "request": "launch",
       "cwd": "${workspaceFolder}",
-      "runtimeExecutable": "yarn",
-      "runtimeArgs": ["run", "debug-react-exp", "dev", "test/e2e/app-dir/app"],
+      "runtimeExecutable": "pnpm",
```

Comment:
`yarn` is still used in [various other scripts](https://github.com/vercel/next.js/search?q=yarn+path%3A%2Fscripts), do those need updating also?

Reply:
Yes, we would ideally use a single package manager everywhere. :+1: Simply replacing all the references might not work, but PRs are welcome! :pray: